test_common/file_objects/et3_pdf_file.rb

```diff
@@ -12,16 +12,28 @@ def initalize(tempfile)
         end
 
         def has_correct_contents_for?(response:, respondent:, representative:, errors: [], indent: 1) # rubocop:disable Naming/PredicateName
-          has_header_for?(respondent, errors: errors, indent: indent) &&
-          has_claimant_for?(response, errors: errors, indent: indent) &&
-          has_respondent_for?(respondent, errors: errors, indent: indent) &&
-          has_acas_for?(response, errors: errors, indent: indent) &&
-          has_employment_details_for?(response, errors: errors, indent: indent) &&
-          has_earnings_for?(response, errors: errors, indent: indent) &&
-          has_response_for?(response, errors: errors, indent: indent) &&
-          has_contract_claim_for?(respondent, errors: errors, indent: indent) &&
-          has_representative_for?(representative, errors: errors, indent: indent) &&
-          has_disability_for?(representative, errors: errors, indent: indent)
+          has_header_for?(respondent, errors: errors, indent: indent)
```

Comment:
Are you sure this is right ?  The has_xxxxxx_for? methods return a boolean and add to the errors collection so any failures are all reported together.
Here you are ignoring all but the final has_____for? call so you could end up with incorrect test failures or passes.

One way you could do it without the ugly '&&'s is to return the value of errors.empty? right at the end of this method so the method returns true if none of the methods have added any errors